powermate.js

```diff
@@ -186,7 +186,7 @@ PowerMate.prototype._parseRead = function(error, data) {
     return;
   } else if (error) {
     if (error.message == "could not read from HID device") {
-      process.nextTick(function() {
+      process.nextTick(()=>{
```

Comment:
Could this be changed to a `function() { }.bind(this)` instead? To maintain compatibility with older Node.js releases.

@brnndnn thanks for the PR :D